tests/testthat/test-shinytest2-init.R

```diff
@@ -8,6 +8,65 @@ testthat::test_that("e2e: teal app initializes with no errors", {
   app$stop()
 })
 
+testthat::test_that("e2e: teal app initializes with sessionInfo modal", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = example_module(label = "Example Module")
+  )
+  app$wait_for_idle(timeout = default_idle_timeout)
+
+  # Check if button exists.
+  button_selector <- "#teal-sessionInfo-button"
+  testthat::expect_equal(
+    app$get_text(button_selector),
+    "Session Info"
+  )
+
+  app$click(selector = button_selector)
+  app$wait_for_idle(timeout = default_idle_timeout)
+
+  # Check header and title content.
+  testthat::expect_equal(
+    app$get_text("#shiny-modal > div > div > div.modal-header > h4"),
+    "SessionInfo"
+  )
+  testthat::expect_equal(
+    app$get_text("#teal-sessionInfo-copy_button1"),
+    "Copy to Clipboard"
+  )
+  testthat::expect_equal(
+    app$get_text("#shiny-modal > div > div > div.modal-body > div > button:nth-child(2)"),
+    "Dismiss"
+  )
```

Comment:
I think those buttons are duplicated, since they exist in the footer already : /
Raised an issue to fix that in here https://github.com/insightsengineering/teal.widgets/issues/233

Reply:
that is actually a feature:) https://github.com/insightsengineering/teal.widgets/issues/233#issuecomment-1994511156